datahub-web-react/src/alchemy-components/components/Editor/EditorTheme.tsx

```diff
@@ -85,6 +86,7 @@ export const EditorContainer = styled.div`
 
         a {
             font-weight: 500;
+            color: ${colors.violet[500]};
```

Comment:
can we do `colors.primary` here instead? i'm trying to push for that everywhere we intend to use our primary color instead of violet